src/interactions/hoverInteraction.ts

```diff
@@ -45,14 +46,20 @@ export module Interaction {
 
     public _anchor(component: Hoverable, hitBox: D3.Selection) {
       super._anchor(component, hitBox);
-      this._dispatcher = new Dispatcher.Mouse(this._hitBox);
+      this._dispatcher = Dispatcher.Mouse.getDispatcher(<SVGElement> (<any> this._componentToListenTo)._element.node());
```

Comment:
What's with the any casting / casting here?

Is _element private/protected?  I remember @jtlan asking that element be exposed